tests/frontend/test_tdmprogram.py

```diff
@@ -34,15 +37,15 @@ def singleloop(r, alpha, phi, theta, copies, shift="default", hbar=2):
     Returns:
         (list): homodyne samples from the single loop simulation
     """
-
+    sf.hbar = 2.0
```

Comment:
This should be removed, and instead all tests that call it should monkeypatch the value of `hbar` (to avoid this global variable change from affecting other files)